feel/feelfmi/fmu.cpp

```diff
@@ -58,13 +67,22 @@ int FMU::load( std::string _path )
 
     if ( boption(_name="fmu.display-variables-info", _prefix=M_prefix) )
         M_model->printVariablesInfo();
+    if ( M_export_list )
+    {
+        M_model->setExportList( M_export_list );
+        M_model->setExportDirectory( M_export_directory );
+    }
 
     return 1;
 }
 
 
 void FMU::simulate( double t_init, double t_final, double tolerance )
 {
+    if ( t_init<0 )
+        t_init = doption( _name="fmu.time-initial", _prefix=M_prefix );
```

Comment:
define member variable from a constructor to hold `fmu.time-initial`value. do not use `<>option` functions in a member function.